app-typescript/components/ToggleBox/index.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import {SimpleToggleBox} from './SimpleToggleBox';
+import {CustomHeaderToggleBox} from './CustomHeaderToggleBox';
+
+export interface IPropsWithLabel {
```

Comment:
interface naming should match the variant. If variant is `simple`, interface should be `IPropsSimple`